includes/REST/PaymentController.php

```diff
@@ -73,10 +76,10 @@ public function payment_permissions_check() {
      *
      * @since 1.0.0
      *
-     * @return void
+     * @return \WP_Error|\WP_HTTP_Response|\WP_REST_Response
      */
-    public function get_avaible_gateways( $request ) {
-        $available_gateways = \WePOS::init()->available_gateway();
+    public function get_available_gateways( $request ) {
+        $available_gateways = GatewayManager::available_gateway();
```

Comment:
Calling `\WePOS::init()->available_gateway();` was the incorrect way of doing things. But so is `GatewayManager::available_gateway()`. We use this pattern everywhere, in Dokan, in WooCommerce as well. So we should be able to call like this: `wepos()->gateways->available_gateway()`. So we don't need to make them static functions and it ensures the class will be executed only once on plugin intialization.

So initialize the class on `WePOS -> init_classes` method and call accordingly.